app/screens/login_options/sso_options.tsx

```diff
@@ -0,0 +1,177 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
This file allows adding SSO options and removes all of the `<SSO_Option>.tsx` files.